ipaplatform/base/paths.py

```diff
@@ -102,6 +102,7 @@ class BasePathNamespace(object):
     SMB_CONF = "/etc/samba/smb.conf"
     LIMITS_CONF = "/etc/security/limits.conf"
     SSH_CONFIG = "/etc/ssh/ssh_config"
+    SSH_CONFIG_DIR = "/etc/ssh"
```

Comment:
Can you please move it one line up, list of path is sorted alphabetically by path, not by variable name